### ISSUE TITLE
How to Disable Swagger UI and openapi.json in Cog

### ISSUE THREAD

**Issue by kosukekurimoto** [replicate/cog#1986]
Hi

I hope this message finds you well. I am currently exploring Cog and would like to understand how to disable the Swagger UI and openapi.json endpoints for security purposes.

Could you please provide guidance or recommended steps to achieve this configuration? Any help or documentation you could point me to would be highly appreciated.

Thank you for your assistance and for your continued work on this project!

**Comment by mcam10** [comment]
Yeah this would need to done on the fastapi side.. unless theres a way in cog to provide overrides? 
https://fastapi.tiangolo.com/tutorial/metadata/#openapi-url @kosukekurimoto

**Comment by markphelps** [comment]
Closing this issue as stale since there has been no activity for over 6 months and no further details were provided. If you still need help with disabling Swagger UI/openapi.json endpoints, or if you believe this requires a new feature in Cog, please open a new issue or discussion with details about your use case and environment.
